extensions/projection/README.md

```diff
@@ -40,43 +40,70 @@ The `proj` prefix is short for "projection", and is not a reference to the PROJ/
 If the data does not have a CRS, such as in the case of non-rectified imagery with Ground Control
 Points, proj:projjson should be set to null. It should also be set to null if a CRS exists, but for which
 a PROJJSON string does not exist. The schema for this object can be found [here](https://proj.org/schemas/v0.2/projjson.schema.json).
 
-**proj:geometry** - A Polygon object representing the footprint of this item, formatted according the Polygon
+#### proj:geometry
+
+A Polygon object representing the footprint of this item, formatted according the Polygon
 object format specified in [RFC 7946, sections 3.1.6](https://tools.ietf.org/html/rfc7946), except not necessarily
 in EPSG:4326 as required by RFC7946.  Specified based on the `proj:epsg`, `proj:projjson` or `proj:wkt2` fields (not necessarily EPSG:4326).
 Ideally, this will be represented by a Polygon with five coordinates, as the item in the asset data CRS should be
 a square aligned to the original CRS grid.
 
-**proj:bbox** - Bounding box of the assets represented by this item in the asset data CRS. Specified as 4 or 6
+#### proj:bbox
+
+Bounding box of the assets represented by this item in the asset data CRS. Specified as 4 or 6
 coordinates based on the CRS defined in the `proj:epsg`, `proj:projjson` or `proj:wkt2` fields.  First two numbers are coordinates
 of the lower left corner, followed by coordinates of upper right corner, , e.g., \[west, south, east, north],
-\[xmin, ymin, xmax, ymax], \[left, down, right, up], or \[west, south, lowest, east, north, highest]. The length of the array must be 2*n where n is the number of dimensions. The array contains all axes of the southwesterly most extent followed by all axes of the northeasterly most extent specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). When using 3D geometries, the elevation of the southwesterly most extent is the minimum elevation in meters and the elevation of the northeasterly most extent is the maximum in meters.
+\[xmin, ymin, xmax, ymax], \[left, down, right, up], or \[west, south, lowest, east, north, highest]. The length of the array must be 2\*n where n is the number of dimensions. The array contains all axes of the southwesterly most extent followed by all axes of the northeasterly most extent specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). When using 3D geometries, the elevation of the southwesterly most extent is the minimum elevation in meters and the elevation of the northeasterly most extent is the maximum in meters.
+
+#### proj:centroid
+
+Coordinates representing the centroid of the item in the asset data CRS.  Coordinates are defined in latitude and longitude, even if 
+the data coordinate system does not use lat/long. This is useful as the projection from the native CRS into GeoJSON (lat/long) can
```

Comment:
@philvarner - is this additional information correct? I was quite confused by the idea of the centroid of the asset data in CRS that needed to be represented in lat/long, but @matthewhanson explained to me what I tried to explain. But he wasn't totally sure. Just want to help people like me to understand why you would use this.

Reply:
I'm not sure the warping explanation is correct -- I'll look into that. In our implementation, we added this for a very practical reason -- in Elasticsearch, a field with an object containing `lat` and `lon` can be put on a Kibana Map. There may be other similar uses.

Reply:
on further reflection, the warping explanation is correct -- the centroid will be in a different place after reprojection to 4326. (though, I'm not entirely sure in my impl that I'm not reprojecting and taking the centroid instead of taking the centroid and reprojecting it. 😬 )

Reply:
Ah, so you just did it so you'd have a centroid in lat/long. I think the thing that confused me was the description said: 'Coordinates representing the centroid of the Item in the asset CRS'. And it directly followed proj:bbox, where the point of that *is* to have the bbox in the asset. If the main case is just to have a lat/long centroid I think maybe we leave out the emphasis on the 'asset crs'. Like have the description be 'Coordinates representing the centroid of the Item, in lat/long', and the detail be about how having the centroid lets you easily plot things on less sophisticated mapping systems. And I'd mention last that it should be in the CRS / before projection.

The way it was written definitely had me overthinking the use case, and it sounds like the use case is much simpler - get a lat/lon for kibana.

Reply:
Went ahead and just made the changes, as this makes a lot more sense. I was just trying to figure out the purpose of the field and trying to explain it, and missed the obvious one.